src/probnum/diffeq/odefiltsmooth/odefiltsmooth.py

```diff
@@ -67,76 +65,71 @@ def probsolve_ivp(
     extended Kalman smoothing (EKS1), and
     unscented Kalman smoothing (UKS).
 
+    For adaptive step-size selection of ODE filters, we implement the
+    scheme proposed by Schober et al. (2019), and further examined by Bosch et al (2021),
+    where the local error estimate is derived from the local, calibrated
+    uncertainty estimate.
+
     Arguments
     ---------
-    ivp : IVP
-        Initial value problem to be solved.
-    step : float
-        Step size :math:`h` of the solver. This defines the
-        discretisation mesh as each proposed step is equal to :math:`h`
-        and all proposed steps are accepted.
-        Only one of out of ``step`` and ``tol`` is set.
-    tol : float
-        Tolerance :math:`\\varepsilon` of the adaptive step scheme.
-        We implement the scheme proposed by Schober et al., accepting a
-        step if the absolute as well as the relative error estimate are
-        smaller than the tolerance,
-        :math:`\\max\\{e, e / |y|\\} \\leq \\varepsilon`.
-        Only one of out of ``step`` and ``tol`` is set.
-    which_prior : str, optional
-        Which prior is to be used. Default is an IBM(1), further support
-        for IBM(:math:`q`), IOUP(:math:`q`), Matern(:math:`q+1/2`),
-        :math:`q\\in\\{1, 2, 3, 4\\}` is provided. The available
-        options are
-
-        ======================  ========================================
-         IBM(:math:`q`)         ``'ibm1'``, ``'ibm2'``, ``'ibm3'``,
-                                ``'ibm4'``
-         IOUP(:math:`q`)        ``'ioup1'``, ``'ioup2'``, ``'ioup3'``,
-                                ``'ioup4'``
-         Matern(:math:`q+0.5`)  ``'matern32'``, ``'matern52'``,
-                                ``'matern72'``, ``'matern92'``
-        ======================  ========================================
-
+    f :
+        ODE vector field.
+    t0 :
+        Initial time point.
+    tmax :
+        Final time point.
+    y0 :
+        Initial value.
+    df :
+        Jacobian of the ODE vector field.
+    adaptive :
+        Whether to use adaptive steps or not. Default is `True`.
+    atol : float
+        Absolute tolerance  of the adaptive step-size selection scheme.
+        Optional. Default is ``1e-4``.
+    rtol : float
+        Relative tolerance   of the adaptive step-size selection scheme.
+        Optional. Default is ``1e-4``.
+    step :
+        Step size. If atol and rtol are not specified, this step-size is used for a fixed-step ODE solver.
+        If they are specified, this only affects the first step. Optional.
+        Default is None, in which case the first step is chosen as :math:`0.01 \cdot |y_0|/|f(t_0, y_0)|`.
+    algo_order
```

Comment:
What about `alg_order`, or even just `order`? I would slightly prefer those two, but feel free to pick whichever you prefer.

Reply:
Thanks for the suggestion! If you dont mind, I'd prefer to keep it that way. Even though my opinion is not strong by any means, here are my thoughts:
* `order` crops up way to often already in my mind. I like that `alg(o)_order` is much clearer in this context, even for people that have no idea about ODE filters
* I slightly prefer "algo" over "alg" (and, in order to pretend that I have a real reference for why this is better: https://stackoverflow.com/questions/5372129/short-name-for-algorithm, though this "reference" should be taken with a grain of salt :D)